### PR TITLE
Move logs icons to a menu

### PR DESCRIPTION
**Pull request by feloy**
Here is my proposition to fix the issue #3754.

In the logs view, the toggle buttons are instead placed in a floating menu.

### CODE REVIEW

**Comment by jeefy** [comment]
This looks awesome and I love the direction.

Couple small thoughts:
- Whenever you click inside the menu popup (anywhere, not just a checkbox), the menu closes.
- Also, whenever you click the text associated with a checkbox, said checkbox doesn't get checked.

If you can address those two behaviors this should get merged in quick. 

/assign

File: src/app/frontend/logs/template.html [hunk 1]
```diff
@@ -155,3 +110,42 @@
...
+                color="primary"
+                [checked]="logService?.getInverted()"
+                (change)="onTextColorChange()">
+    Invert colors
+  </mat-checkbox>
+  <mat-checkbox class="mat-menu-item"
+                color="primary"
+                [checked]="logService.getCompact()"
+                (change)="onFontSizeChange()">
+    Small font size
```
Comment (maciaszczykm) [inline src/app/frontend/logs/template.html]: It should change after clicking the element.

**Comment by jeefy** [comment]
@feloy There are some formatting nits that are causing the Travis job to fail.

Please run `npm run check` locally before committing/pushing. This should show these formatting issues ahead of time. :) Let me know if you have any questions.

When these nits get fixed this looks good to merge!

**Comment by feloy** [comment]
Hi @jeefy 
It seems that the formatting issues were included in the master with the patch updating gts to v1.0.0 (#3761)
I would prefer to create a pull request to resolve these issues separately from this PR.

**Comment by maciaszczykm** [comment]
Please rebase/merge master to fix the issues with the CI job. Sorry for the inconvenience.

**Comment by jeefy** [comment]
@feloy There's one more conflict that needs resolving. Everything else lgtm so hoping to merge this soon.

Thanks!

**Pull request merged**